src/pages/Shop/Shop.js

```diff
@@ -1,11 +1,189 @@
+  const [isOpen, setIsOpen] = useState(false);
+
+  const onClickCategory = () => {
+    setIsOpen(!isOpen);
+  };
+
   return (
-    <div>
-      <div>Shop Page</div>
-    </div>
+    <ShopWrapper>
+      <ShopAsideBox>
+        <AsideTitle>필터</AsideTitle>
+        {SHOP_CATEGORY.map(({ id, title, subTitle, subCategory }) => {
+          return (
+            <>
+              <AsideCategoryListBox key={id}>
+                <AsideCategoryTitleBox>
+                  <AsideCategoryTitle>{title}</AsideCategoryTitle>
+                  <AsideCategorySubTitle>{subTitle}</AsideCategorySubTitle>
+                </AsideCategoryTitleBox>
+                <AsidePlusBtn onClick={onClickCategory}>+</AsidePlusBtn>
+              </AsideCategoryListBox>
+              <AsideOpenCategoryList
+                key={subCategory.id}
+                subCategory={subCategory}
+              >
+                {subCategory.map(({ subId, text }) => {
+                  return (
+                    <AsideOpenCategoryItem
+                      key={subId}
+                      className={`${isOpen ? 'open' : 'close'}`}
+                    >
+                      <AsideOpenCategoryInput id="option" type="checkbox" />
+                      <AsideOpenCategoryLabel for="option">
+                        {text}
+                      </AsideOpenCategoryLabel>
+                    </AsideOpenCategoryItem>
+                  );
+                })}
+              </AsideOpenCategoryList>
+            </>
+          );
+        })}
+      </ShopAsideBox>
+      <ShopProduct />
+    </ShopWrapper>
   );
 };
 
 export default Shop;
+
+const ShopWrapper = styled.div`
+  width: 80%;
+  display: flex;
+  justify-content: space-between;
+  margin: 0 auto;
+  padding: 131px 40px 80px;
+`;
+
+const ShopAsideBox = styled.div`
+  width: 35%;
+  margin-right: 25px;
+`;
+
+const AsideTitle = styled.h1`
+  font-size: 15px;
+  font-weight: bold;
+  margin-bottom: 20px;
+`;
+
+const AsideCategoryListBox = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  border-bottom: ${({ theme }) => theme.globalBoardStyle};
+`;
+
+const AsideCategoryTitleBox = styled.div`
+  padding: 20px 0px;
+`;
+
+const AsideCategoryTitle = styled.h2`
+  font-size: 13px;
+  font-weight: bold;
+  margin-bottom: 8px;
+`;
+
+const AsideCategorySubTitle = styled.h3`
+  font-size: 13px;
+  color: ${({ theme }) => theme.mainBrandGray05};
+`;
+
+const AsidePlusBtn = styled.span`
+  font-size: 25px;
+  font-weight: lighter;
+  cursor: pointer;
+  color: ${({ theme }) => theme.mainBrandGray05};
+`;
+
+const AsideOpenCategoryList = styled.ul`
+  border-bottom: ${({ theme }) => theme.globalBoardStyle};
+`;
+
+const AsideOpenCategoryItem = styled.li`
+  &.open {
+    max-height: 500px;
+    overflow: hidden;
+    transition: max-height ease-in-out 2s 0s;
+  }
+  &.close {
+    max-height: 0px;
+    overflow: hidden;
+    transition: max-height ease-in-out 1s 0s;
+  }
+`;
```

Comment:
close 상태가 default 상태 이기 때문에 open 일 때만 핸들링 하면 더 좋을 것 같습니다!

```suggestion
const AsideOpenCategoryItem = styled.li`
  max-height: 0;
  overflow: hidden;
  transition: max-height ease-in-out 1s 0s;
  
  &.open {
    max-height: 500px;
  }
`;
```

Reply:
넵 감사합니다 !! 수정 완료했습니다 :)